packages/cli/package.json

```diff
@@ -93,6 +93,7 @@
     "@types/which": "1.3.2",
     "@types/write-json-file": "2.2.1",
     "@types/yauzl-promise": "2.1.0",
+    "@vercel-internals/types": "workspace:*",
```

Comment:
Didn't this syntax break last time?

Reply:
I think so. But I forget what it broke so I wanted to see if we could fix that this time around.

Reply:
This broke the published package, if I recall correctly